pytest_splunk_addon/splunk.py

```diff
@@ -244,20 +279,49 @@ def splunk_external(request):
     return splunk_info
 
 
-
 @pytest.fixture(scope="session")
 def splunk_rest_uri(splunk):
     """
     Provides a uri to the Splunk rest port
     """
     splunk_session = requests.Session()
     splunk_session.auth = (splunk["username"], splunk["password"])
-    uri = f'https://{splunk["host"]}:{splunk["port"]}/'
+    uri = f'https://{splunk["host"]}:{splunk["port_hec"]}/'
     LOGGER.info("Fetched splunk_rest_uri=%s", uri)
 
     return splunk_session, uri
 
 
+@pytest.fixture(scope="session")
+def splunk_hec_uri(request, splunk):
```

Comment:
looks like the function can be refactored into splunk_hec_uri_raw as they both look very similar?

Reply:
yes this could be refactored the URI could is different it would be about the same total lines of code